admin-part/src/main/kotlin/coverage/Ext.kt

```diff
@@ -31,10 +31,10 @@ internal fun <T> List<T>.slice(probeRange: ProbeRange): List<T> = slice(probeRan
 internal fun Count.percentage(): Double = covered percentOf total
 
 internal fun Count?.arrowType(other: Count): ArrowType = this?.run {
-    (this - other).first.sign.toArrowType()
+    (this.subtraction(other)).first.sign.toArrowType()
 } ?: ArrowType.UNCHANGED
 
-internal operator fun Count.minus(other: Count): Pair<Long, Long> = takeIf { other.total > 0 }?.run {
+internal infix fun Count.subtraction(other: Count): Pair<Long, Long> = takeIf { other.total > 0 }?.run {
```

Comment:
why is minus not suitable here?

Reply:
since the count it's Long now , we can't override operator for it . if you are talking about naming minus is kotlin function and if we name our function with the same name we will use kotlin fun ((